build_scripts/build_usd.py

```diff
@@ -2058,6 +2064,10 @@ def InstallUSD(context, force, buildArgs):
                       help="Build OpenImageIO plugin for USD")
 subgroup.add_argument("--no-openimageio", dest="build_oiio", action="store_false",
                       help="Do not build OpenImageIO plugin for USD (default)")
+if MacOS():
+    group.add_argument("--imageio", dest="build_imageio", action="store_true", 
+                      default=False,
+                      help="Build ImageIO plugin for USD (Uses ImageIO.framework). Overrides OpenImageIO")
```

Comment:
"Overrides" is a little vague, since it could refer to runtime behavior.  Instead, could we say "Disables building OpenImageIO" ?